sufia-models/app/models/sufia/works/curation_concern/with_basic_metadata.rb

```diff
@@ -0,0 +1,35 @@
+# Basic metadata for all Works
+# Required fields:
```

Comment:
Are we saying that keywords and creator will no longer be required?  The documentation here is in conflict with a longstanding Sufia feature.